ga4gh/datamodel/ontologies.py

```diff
@@ -32,7 +33,27 @@ def getId(self, name):
     def getName(self, id_):
         return self._nameIdMap[id_]
 
+    def getGaTermByName(self, name):
```

Comment:
What does GaTerm stand for here? It would be nice to have some docstrings here for these new functions, as it's not entirely obvious what's going on.